torch-neuronx/training/tp_dp_gpt_neox_hf_pretrain/tp_dp_gpt_neox_6.9b_hf_pretrain.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+#############################################
+# User defined parameters and env vars
+
+export NEURON_CC_FLAGS="--model-type=transformer --enable-experimental-O1 --enable-internal-call-graph --enable-saturate-infinity"
```

Comment:
We don't need `enable-internal-call-graph` as that is bundled inside experimental-o1